<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-KGLKBPSV" failures="2" tests="2" name="Test" time="22.919" errors="0" timestamp="2025-03-09T23:56:28 IST">
  <testcase classname="com.dtproviders.DataProvidersEx" name="loginDataCheck" time="4.888">
    <failure type="java.lang.AssertionError" message="incorrect message is displayed.... expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: incorrect message is displayed.... expected [false] but found [true]
at com.dtproviders.DataProvidersEx.loginDataCheck(DataProvidersEx.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginDataCheck -->
  <testcase classname="com.dtproviders.DataProvidersEx" name="loginDataCheck" time="8.717">
    <failure type="java.lang.AssertionError" message="incorrect message is displayed.... expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: incorrect message is displayed.... expected [false] but found [true]
at com.dtproviders.DataProvidersEx.loginDataCheck(DataProvidersEx.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginDataCheck -->
</testsuite> <!-- Test -->
